### PR TITLE
Fix trades bug

### PR DESCRIPTION
**Pull request by sydneynotthecity**
Closes bug [GH260](https://github.com/stellar/hubble/issues/260)

### CODE REVIEW

File: internal/transform/trade.go [hunk 1]
```diff
@@ -106,9 +106,9 @@ func TransformTrade(operationIndex int32, operationID int64, transaction ingest.
 		}
 
 		if BuyingOffer != nil {
-			outputBuyingOfferID = null.IntFrom(int64(claimOffer.OfferId()))
+			outputBuyingOfferID = null.IntFrom(int64(BuyingOffer.OfferId))
```
Comment (Shaptic) [inline internal/transform/trade.go:109]: Copy-paste strikes again eh?

File: internal/transform/trade.go [hunk 2]
```diff
@@ -106,9 +106,9 @@ func TransformTrade(operationIndex int32, operationID int64, transaction ingest.
...
 		} else {
-			outputBuyingOfferID = null.IntFrom(toid.EncodeOfferId(uint64(operationID), toid.TOIDType))
+			outputBuyingOfferID = null.IntFrom(toid.EncodeOfferId(uint64(operationID)+1, toid.TOIDType))
```
Comment (Shaptic) [inline internal/transform/trade.go:111]: Why +1? Maybe worth a `// comment`

**Review by Shaptic — APPROVED** [review]
I don't really know what I'm reviewing but the code LGTM besides one question :arrow_down:

**Review by erika-sdf — APPROVED** [review]

File: internal/transform/trade.go (hunk 1 above)
Comment (sydneynotthecity) [reply internal/transform/trade.go:109]: In the worst possible way

File: internal/transform/trade.go (hunk 2 above)
Comment (sydneynotthecity) [reply internal/transform/trade.go:111]: it's a quirk with the `ingest` package where the `operation_id` is off by 1. I'll add a comment

**Pull request merged**